filters/cloudychat/cloudychat.go

```diff
@@ -146,7 +147,10 @@ func ChatFilter(handler *potoq.Handler, rawPacket packets.Packet) error {
 		MessageHook(handler, "chat", msg)
 	}
 
-	log.Info(handler.Nickname + ": " + input.Message)
+	chat_log.WithFields(logrus.Fields{
+		"nickname": handler.Nickname,
+		"uuid": handler.UUID,
+	}).Info(input.Message)
```

Comment:
You can add a function to the handler that returns logrus.Fields. Example:
```go
func (handler *Handler) LoggingFields() logrus.Fields {
	return logrus.Fields{
		"username": handler.Nickname,
		"uuid": handler.UUID,
	}
}
```
and then:
```suggestion
	chat_log.WithFields(handler.LoggingFields()).Info(input.Message)
```

Reply:
To jest logging od chatu, handler już ma Log() logrus.FieldLogger, ale wrzuca też pole które są troche zbędne w kontekscie chatu imho

Reply:
Może lepiej jest zrobić jakis helper `cloudychat.chatFields(handler) logrus.FieldLogger`?

Reply:
Mozna, aczkolwiek te same pola sa uzywane w handler.Log(). Mozna by bylo zmienic handler.Log() na cos takiego:
```go
func (handler *Handler) Log() logrus.FieldLogger {
	return logrus.WithFields(handler.LoggingFields()).WithFields(logrus.Fields{
		"upstream": handler.UpstreamName,
		"downstreamAddr": handler.DownstreamAddr,
	})
}
```

Reply:
Nazwa ```LoggingFields``` też jest raczej do zmienia, ale to już zostawiam wam.

Reply:
IMHO w ten sposób komplikujesz to tylko, żeby niby nie powtarzać kilku linijek. Lepiej powtarzać i mieć czytelny kod niż 20 magicznych helperów